src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -8226,7 +8722,13 @@ namespace HVACVariableRefrigerantFlow {
             VRFTU(VRFTUNum).CalcVRF(VRFTUNum, FirstHVACIteration, PartLoadRatio, ActualOutput, OnOffAirFlowRatio, SuppHeatCoilLoad);
         }
 
-        PLRResidual = (ActualOutput - QZnReq) / QZnReqTemp;
+        if (setPointControlled) {
```

Comment:
Modified existing iteration function to use either SP or load controlled calculations.

Reply:
I think I need to fix this since the SP can be at the individual coil outlet or the TU outlet node.